### PR TITLE
Security utility towel

### PR DESCRIPTION
**Pull request by JORJ949**
## About the PR 
Adds a security towel to the security apparel vendor to clean off the blood they will likely get covered in at some point
image

## Why's this needed? 
Encourages cleaning yourself after ~~brutally murdering the clown~~ valiantly fighting traitors!

## Changelog 

```changelog
(u)JORJ949
(*)Added security towels found in the security apparel vendor
```

### CODE REVIEW

File: code/obj/submachine/weapon_vendor.dm [hunk 1]
```diff
@@ -409,6 +413,12 @@
 	path = /obj/item/device/prisoner_scanner
 	description = "A device used to scan in prisoners and update their security records."
 
+/datum/materiel/utility/sectowel
+	name = "Security Towel"
+	path = /obj/item/cloth/towel/security
+	description = "A Towel used to clean off the clow- criminal blood!"
```
Comment (Glamurio) [inline code/obj/submachine/weapon_vendor.dm:419]: I know it's minor, but don't capitalize towel here.

**Review by Glamurio — CHANGES_REQUESTED** [review]

File: code/obj/submachine/weapon_vendor.dm (hunk 1 above)
Comment (Glamurio) [reply code/obj/submachine/weapon_vendor.dm:419]: That made me chuckle. I was speaking of the description.

Comment (JORJ949) [reply code/obj/submachine/weapon_vendor.dm:419]: >:(

**Comment by Valtsu0** [comment]
I know sec has the same color scheme, but that looks a lot like a syndie item. Maybe its fine?

**Comment by JORJ949** [comment]
> I know sec has the sameish color scheme, but that looks a lot like a syndie item. Maybe its fine?

Palette is directly taken from the secoff uniform. Thats just them having the same colour scheme

**Review by TobleroneSwordfish — APPROVED** [review]
Seems cool, it's a pretty low "value" utility purchase but I'm sure people will like it anyway!

**Comment by JORJ949** [comment]
> Seems cool, it's a pretty low "value" utility purchase but I'm sure people will like it anyway!

If the vendor system supports making things with different costs this could be a great way to balance it but currently the donuts are by far the most popular option and for good reason, they have saved me many times and often save a lengthy medbay trip. Other things like the RecordTrak and Forensic scanner can often be found elsewhere without costing a utility credit.

**Comment by Sord213** [comment]
IMO I'd personally rather see the items in the vendor be actually useful for security work. This seems like it'd be better in the clothing vendor to me.

**Pull request merged**